lib/Dialect/ArithExt/IR/ArithExtOps.td

```diff
@@ -42,6 +43,13 @@ def ArithExt_MulOp : ArithExt_BinaryOp<"mul", [Commutative]> {
   }];
 }
 
+def ArithExt_MacOp : ArithExt_Op<"mac", [SameOperandsAndResultType, Pure, ElementwiseMappable]> {
+  let summary = "modular multiplication-and-accumulation operation";
```

Comment:
```suggestion
  let summary = "modular multiplication-and-accumulation operation";
  
  let description = [{
    The modular multiplication and accumulation operation computes
    
    `arith_ext.mac x, y, acc {modulus = q} = (x * y) + acc \mod q`
  }];
```

Reply:
We have mathmode on the website if you want to use it

`arith_ext.mac x, y, z {modulus = q}` computes $(x * y) + z \mod q$